File: Data Structures & Algorithms/two-integer-sum/submission-11.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> mapping = new HashMap<>();

        for(int i = 0; i < nums.length; i++){
            int calc = target - nums[i];
            if(mapping.containsKey(calc)) return new int[]{mapping.get(calc), i};
            mapping.put(nums[i], i);

        }
        return new int[]{}; 
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-11.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        setversion = set(nums)
        length = 0
        for num in nums:
            if (num - 1) not in setversion: 
                while (num + length) in setversion:
                    length += 1 
                

        return length 